packages/components/tooltip/src/content.vue

```diff
@@ -102,7 +102,7 @@ export default defineComponent({
     })
 
     const shouldRender = computed(() => {
-      return unref(persistentRef) ? true : unref(open)
+      return persistentRef
```

Comment:
There is a problem with changes that return persistentRef, this will make shouldRender always true

Reply:
return unref(persistentRef) ? true : ref(open)